client/src/main.rs

```diff
@@ -600,8 +603,44 @@ fn read_shard(matches: &ArgMatches<'_>) -> StdResult<ShardIdentifier, codec::Err
         },
     }
 }
+
+fn send_direct_request_encoded(
+    matches: &ArgMatches<'_>,
+    operation_call: TrustedOperation,
+) -> Option<Vec<u8>> {
+    let operation_call_encoded = operation_call.encode();
+
+    let shard = read_shard(matches).unwrap();
+
+    // compose jsonrpc call
+    let data = Request {
+        shard,
+        cyphertext: operation_call_encoded,
+    };
+
+    let rpc_method_str_option = get_rpc_function_name_from_top(&operation_call);
+    if let None = rpc_method_str_option {
+        println!("[Error]: This type of TrustedOperation is not supported")
+    }
+
+    let rpc_method_str = rpc_method_str_option.unwrap();
+
+    debug!("Got trusted operation for RPC method {}", rpc_method_str);
+
+    let direct_invocation_call = RpcRequest {
+        jsonrpc: "2.0".to_owned(),
+        method: rpc_method_str,
+        params: data.encode(),
+        id: 1,
+    };
+
+    let direct_api = get_worker_api_direct(matches);
+
+    send_direct_request(direct_invocation_call, direct_api)
+}
```

Comment:
We currently only encode our TrustedOperations, we don't encrypt them anymore.

Reply:
correct, as agreed. but they have to be signed

Reply:
Yes, the signing happens in the client. In the worker we check the signature.

Cyphertext ist a field in 'Request', which is pulled in from the pallet-substratee-registry, so it probably cannot be changed so easily?

Reply:
I suggest introducing a new type `DirectRequest` which is unencryped, possibly in polkadex_primitives....